### DISCUSSION TITLE
Upgrade httr to httr2 planned?

### DISCUSSION THREAD

**Discussion by salvafern** [eblondel/ows4R#103] [General]
Hi @eblondel

Once again many thanks for this excellent package. This is my first option when having to deal with OGC in R, even as dependency for other packages. 

I think this has been mentioned before but can't find an issue or discussion about the topic. **Are you planning to upgrade the dependency on `httr` to `httr2`?**

Triggered by https://github.com/lifewatch/mregions2/issues/17

**Comment by eblondel** [comment]
Hi @salvafern no, it's not in immediate plans. 
This will be part of a broader assessment to see pros/cons and measure effort/costs implications to move from httr to httr2 in all SDI-related packages (including ows4R, but also geonapi, geosapi, etc).

**Comment by salvafern** [comment]
Ok perfectly understandable. Thanks for the quick reply!
